### DISCUSSION TITLE
About multi-classification issue.

### DISCUSSION THREAD

**Discussion by XieXiao1997** [Project-MONAI/MONAI#3230] [Q&A] [answered]
When I tried to mimic the 3D classification in MONAI's example to build a multi-classification model,I had endless problems...like dimension error,target error..It's driving me crazy
Who can give me some advise to do the 3D classify in my own dataset?Thanks...

**Comment by Nic-Ma** [comment, marked as answer]
Hi @demodoom1 ,

Thanks for your feedback.
About 3D classification task, I would suggest referring to this example as start point:
https://github.com/Project-MONAI/tutorials/blob/master/3d_classification/torch/densenet_training_dict.py

Thanks.

**Comment by XieXiao1997** [comment]
Thanks for your help..It is a busy week.I will try it now
